### ISSUE TITLE
Cps/line-length issue

### ISSUE THREAD

**Issue by JDTR75** [SubtitleEdit/subtitleedit#5797]
Hello,

As the title says, I'm having a weird issue here with this setting. When I use "Count all except space", the CPL count is **not** counting the spaces. Spaces should be omitted from the CPS count but not from the CPL count.

I'm using SE 3.6.4 Beta 501, just so you know.

Thanks in advance!

**Comment by niksedk** [comment]
Could you give some more info... screenshots, sample texts etc, thx.

**Comment by JDTR75** [comment]
Sure thing. Here it goes.

"Count all characters" is working fine. Here's and example:
![image](https://user-images.githubusercontent.com/1891941/155890571-a33cafd6-c0f8-459b-9f99-f829b10a4caa.png)

This is what it's doing when you to change to "Count all except space":
![image](https://user-images.githubusercontent.com/1891941/155890599-647af2a3-c8e5-4cdd-8c17-e446107274fb.png)

As you can see, the CPS count is fine but the CPL count is NOT. It's not even flagging the line in red when it's surpassing the 42 characters limit. This is even affecting the "Fix common errors" tool because it's telling you to merge "short" lines when it's not the case.

This is what it should be doing.

This is with full characters:
![image](https://user-images.githubusercontent.com/1891941/155890678-45499217-ff6e-4910-b0e3-f7479b61f8cf.png)

This is the proper "excluding spaces" option:
![image](https://user-images.githubusercontent.com/1891941/155890705-25e94bbc-73bf-4fc3-a803-928f9a0a0446.png)

I hope this is clear enough now, but let me know if you anything else.

Best!

**Comment by niksedk** [comment]
OK, it does what the UI says: works for both line length and cps...
![image](https://user-images.githubusercontent.com/1261331/155891030-55d66202-adc9-4ede-a1f7-7338b764817b.png)

**Comment by JDTR75** [comment]
Yeah, but that shouldn't be the case. It should be like the 3.6.4 stable version.

![image](https://user-images.githubusercontent.com/1891941/155891364-ceeb7bc5-10d9-40a4-b0fd-2ff15a36bca7.png)

**Comment by niksedk** [comment]
This is the correct case for e.g. Korean, but yeah, I broken Latin languages ;)

thx  for the explaining  :)
I better look into something...

**Comment by JDTR75** [comment]
No problem at all. You're welcome :)

**Comment by niksedk** [comment]
How is this?
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.4/SubtitleEditBeta.zip

![image](https://user-images.githubusercontent.com/1261331/155936192-70cb49ee-fcc5-4c2e-882c-8b4d75d1f644.png)

**Comment by JDTR75** [comment]
It's working fine now :)

**Comment by JDTR75** [comment]
Sorry, I know you closed this one. But I didn't want to generate a new issue since it's related to this. It's more of a request than an issue, actually. Is it possible adding this setting in the Cps/line-length?

![image](https://user-images.githubusercontent.com/1891941/156079325-e852be32-ef19-4bb8-9e01-8e17a57db175.png)

Basically, counting spaces as 0.5 as well.

Thanks in advance!

**Comment by niksedk** [comment]
Yes, stuff like this s possible with the new impllementation - but the above is for Korean or japanese, right? It's already fixed...

**Comment by JDTR75** [comment]
Yes, the above is for CJK.
